applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.cpp

```diff
@@ -567,9 +573,9 @@ void IncompressiblePotentialFlowElement<Dim, NumNodes>::CheckWakeCondition() con
 }
 
 template <int Dim, int NumNodes>
-void IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputePotentialJump(ProcessInfo& rCurrentProcessInfo)
+void IncompressiblePotentialFlowElement<Dim, NumNodes>::ComputePotentialJump()
 {
-    const array_1d<double, 3> vinfinity = rCurrentProcessInfo[VELOCITY_INFINITY];
+    const array_1d<double, 3> vinfinity = GetProperties().GetValue(VELOCITY_INFINITY);
```

Comment:
Why in the properties? For me it seems more natural to save the infinite velocity in the process info since as far as I know is a unique magnitude for the entire simulation (correct me if I'm wrong).

Reply:
This is totally true and I think @msandre also commented on this, in fact at the beginning we had it in the properties. The point is that we need to access this variable in many different functions and we did not want to pass rCurrentProcessInfo around everywhere. Does it really matter where it is stored? Shall we change this?

Reply:
the processInfo is intended to be passed everywhere
I would say that it stores "global" variables, whereas the properties only store them for a group of elements or conditions

Reply:
Ok, @rubenzorrilla  @philbucher I agree with what you say so I will change it according to your review.